app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -869,12 +973,14 @@ class PullRequestHelper {
               'oid': lastCommitHash,
               'pushedDate': (dateTime ?? DateTime.now().add(const Duration(hours: -2))).toUtc().toIso8601String(),
               'status': <String, dynamic>{
-                'contexts': lastCommitStatuses.map((StatusHelper status) {
-                  return <String, dynamic>{
-                    'context': status.name,
-                    'state': status.state,
-                  };
-                }).toList(),
+                'contexts': lastCommitStatuses != null
+                    ? lastCommitStatuses.map((StatusHelper status) {
+                        return <String, dynamic>{
+                          'context': status.name,
+                          'state': status.state,
+                        };
+                      }).toList()
+                    : <dynamic>[]
```

Comment:
Is there a test for when a failure occurs in a repo that it can continue on remaining repos?

Reply:
Added a test for that.